src/main/java/pages/LoginPage.java

```diff
@@ -8,43 +8,53 @@
 
 import java.time.Duration;
 
-import static util.Constants.*;
-
 
 public class LoginPage {
 
-    private static final By LOGIN_BTN = By.xpath("//div[@class='HeadBanner-ButtonsWrapper']/a[2]");
+    private static final By LOGIN_BTN = By.xpath("//div[@class='PSHeader-Right']/button");
     private static final By SUBMIT_BTN = By.cssSelector("[type=submit]");
     private static final By LOGIN_FLD = By.name("login");
     private static final By PASSWORD_FLD = By.xpath("//input[@name='passwd']");
     private static final By SIG_IN_BTN = By.id("passp:sign-in");
     private static final By COMPOSE_BTN = By.xpath("//a[@href='#compose']");
-
+    private static final By USER_MENU = By.xpath("//div[@class='PSHeader-User']/div");
+    private static final By LOGOUT_BUTTON = By.xpath("//li/a[contains(@class, 'exit')]");
 
     WebDriver driver;
 
     public LoginPage(WebDriver driver) {
         this.driver = driver;
     }
 
-    public void login(String login, String password) throws InterruptedException {
-        Thread.sleep(1000);//Explicit wait
-        WebElement loginButtonOnStartPage = driver.findElement(LOGIN_BUTTON);
+    public void login(String login, String password) {
+        WebElement loginButtonOnStartPage = driver.findElement(LOGIN_BTN);
         loginButtonOnStartPage.click();
-        WebElement loginInput = driver.findElement(LOGIN_FIELD);
+        WebElement loginInput = driver.findElement(LOGIN_FLD);
         loginInput.sendKeys(login);
-        WebElement submitButton = driver.findElement(SUBMIT_BUTTON);
+        WebElement submitButton = driver.findElement(SUBMIT_BTN);
         submitButton.click();
-        WebElement passwordF = driver.findElement(PASSWORD_FIELD);
+        WebElement passwordF = driver.findElement(PASSWORD_FLD);
         passwordF.sendKeys(password);
-        WebElement loginButton2 = driver.findElement(SIG_IN_BUTTON);
+        WebElement loginButton2 = driver.findElement(SIG_IN_BTN);
         loginButton2.click();
     }
 
+    public void logout() {
+        WebElement user_menu = driver.findElement(USER_MENU);
+        user_menu.click();
+        WebElement logout = driver.findElement(LOGOUT_BUTTON);
+        logout.click();
+    }
+
+    public boolean isButtonDisplayed(By button) {
```

Comment:
call this method isDisplayed() and use specific element. Ex.:
```
public boolean isDisplayed() {
        return sigInButton.isDisplayed();
    }
```

and in the test it will be: `Assert.assertTrue(loginPage.isDisplayed());`